lnwallet/channel.go

```diff
@@ -565,40 +565,189 @@ type htlcView struct {
 	theirUpdates []*PaymentDescriptor
 }
 
-// fetchHTLCView returns all the candidate HTLC updates which should be
-// considered for inclusion within a commitment based on the passed HTLC log
-// indexes.
-func (lc *LightningChannel) fetchHTLCView(theirLogIndex, ourLogIndex uint32) *htlcView {
+// getHTLCView convert HTLC logs into htlcView representation.
+func (lc *LightningChannel) getHTLCView() *htlcView {
 	var ourHTLCs []*PaymentDescriptor
 	for e := lc.ourUpdateLog.Front(); e != nil; e = e.Next() {
 		htlc := e.Value.(*PaymentDescriptor)
+		ourHTLCs = append(ourHTLCs, htlc)
+	}
+
+	var theirHTLCs []*PaymentDescriptor
+	for e := lc.theirUpdateLog.Front(); e != nil; e = e.Next() {
+		htlc := e.Value.(*PaymentDescriptor)
+		theirHTLCs = append(theirHTLCs, htlc)
+	}
+
+	return &htlcView{
+		ourUpdates:   ourHTLCs,
+		theirUpdates: theirHTLCs,
+	}
+}
+
+type EntryHandler func(entry *PaymentDescriptor,
+	isIncoming, isAddEntry bool) (bool, error)
+
+// processHTLCView processes all update entries in both HTLC update logs,
+// producing the new view which is the result of applying processing
+// function to the HTLC view set. Serves as "map", "filter" function
+// for htlcView set.
+func (lc *LightningChannel) processHTLCView(view *htlcView,
```

Comment:
Any particular reason for the name change within the function? (was previously `evaluateHTLCView`)